### ISSUE TITLE
Rule 2ee8b8 ("Visible label is part of accessible name"): adding label text as part of visible label for labelable elements

### ISSUE THREAD

**Issue by shunguoy** [auto-wcag/auto-wcag#2236]
Using label elements to associate text labels with form controls (e.g., radio, checkbox) is common, and may be required by [WCAG](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html).  If a labeled element doesn't have a valid aria-label or aria-labelledby attribute, the text from the label element can be used as an accessible name. However, suppose a labeled element has an aria-label or aria-labelledby attribute. In that case, the accessible name is calculated from the aria-label or aria-labelledby, and this rule should be triggered (with an error) if the label text is not part of the accessible name.  
   For example, 
           <label for="foo">email:</label><input type="email" id='foo' aria-label="E-mail" value="Contact" />

   Also, refer to #2075 for more discussions.
